internal/pkg/initialize/workload_test.go

```diff
@@ -297,7 +298,7 @@ func TestAppInitOpts_createLoadBalancedAppManifest(t *testing.T) {
 
 			wantedPath: "/",
 		},
-		"creates manifest with {app name} as the path if there's another LBWebApp": {
```

Comment:
Just curious, did we ever have this implementation where app name was the path for LBWS?

Reply:
this is legacy code (before v1.0) back when copilot used to call "apps" -> "projects" and "services" -> "apps". I just updated the testcase name to reflect the terminology we've been using